### PR TITLE
Update to Croatian translation

### PR DESCRIPTION
**Pull request by arvvoid**
### Check if this PR fulfills these requirements:
* [X] My changes respect the [Kodi add-on development rules](https://kodi.wiki/view/Add-on_rules)
* [X] I have read the [**CONTRIBUTING**](Contributing.md) document.
* [X] I made sure there wasn't another one [Pull Requests](../../pulls) opened for the same update/change
* [X] I have successfully tested my changes locally

#### Types of changes
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Feature change (non-breaking change which change behaviour of an existing functionality)
- [X] Improvement (non-breaking change which improve functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Description
Update to the Croatian translation

### CODE REVIEW

File: resources/language/resource.language.hr_hr/strings.po [hunk 1]
```diff
@@ -238,32 +251,697 @@ msgstr "Profil"
...
+"The device must:\r\n"
+"Be Netflix certified, support for Widevine Security level L1 and have a 4K display HDCP 2.2"
+msgstr ""
+"Podržava li vaš uređaj 4K standard?\r\n"
+"Uređaj mora:\r\n"
+"Biti Netflix certificiran, imati podršku za Widevine razinu sigurnosti L1 i 4K ekran sa HDCP 2.2 podrškom"
+
+msgctxt "#30157"
+msgid "The InputStream Adaptive settings panel will now be opened, so you have to change this option \"Override HDCP status\" to ON"
+msgstr "Panel prilagodljivih postavki InputStream sada će se otvoriti, morate promijeniti opciju \\ \"Override HDCP status \" u ON"
```
Comment (CastagnaIT) [inline resources/language/resource.language.hr_hr/strings.po]: "\\" unnecessary!?

File: resources/language/resource.language.hr_hr/strings.po [hunk 2]
```diff
@@ -238,32 +251,697 @@ msgstr "Profil"
...
+msgid "Show the trailers"
+msgstr "Pokažite trailere"
+
+msgctxt "#30180"
+msgid "No trailers available"
+msgstr "Nema dostupnih trailera"
+
+msgctxt "#30181"
+msgid "Disable subtitles if there are no forced subtitles streams (Works only if Kodi Player is set to \"Only forced\" subtitles)"
+msgstr "Onemogućite titlove ako nema prisiljenih titlova (Radi samo ako je Kodi Player postavljen na \\ \"Samo prisilni \" titlovi)"
```
Comment (CastagnaIT) [inline resources/language/resource.language.hr_hr/strings.po]: "\\" unnecessary!?

File: resources/language/resource.language.hr_hr/strings.po (hunk 1 above)
Comment (arvvoid) [reply resources/language/resource.language.hr_hr/strings.po]: fixed tnx

File: resources/language/resource.language.hr_hr/strings.po (hunk 2 above)
Comment (arvvoid) [reply resources/language/resource.language.hr_hr/strings.po]: fixed tnx

**Comment by CastagnaIT** [comment]
look good thanks

**Pull request merged**